### ISSUE TITLE
new subcommand `--query`

### ISSUE THREAD

**Issue by onur-ozkan** [lodosgroup/lpm#72]
Implementation of new subcommand `--query` which dumps the target package information on stdout.

e.g., usage: `lpm --inspect $package`

**Comment by odiferousmint** [comment]
Any reasons for why `inspect` instead of `query`? The latter seems to be much more commonly used to `query` the package information. With `query`, you could also use `-q`, `-i` is already being used for `--install`.

If it is just a personal preference, that is fine, though, but I believe using `query` may be more beneficial than `inspect`.

**Comment by onur-ozkan** [comment]
I agree, in terms of user experience, it's fairly better to use "-q/--query". Thanks for the suggestion and explaining reason behind it.
